### ISSUE TITLE
Cannot read property 'fullCount' of undefined

### ISSUE THREAD

**Issue by agrothe** [arangodb/arangodb#2412]
## my environment running ArangoDB
2017-04-04T21:53:49Z [14760] INFO ArangoDB 3.1.12 [win64] 64bit, using VPack 0.1.30, ICU 54.1, V8 5.0.71.39, OpenSSL 1.0.2a 19 Mar 2015

On this operating system: Windows 10 64bit

### this is a web interface-related issue:
I'm using the web interface with this browser:     running on this OS:
- [ X] authentication is enabled?
- [ ] using the cluster?
- [ X] _system database?

These are the steps to reproduce:
1) open the browser on http://127.0.0.1:8529
2) log in as root
3) use database [X ] `_system` [ ] other: 
4) click to ApplicationRole Colletion (create new collection with defaults)
...

Collection only contains this data:

for r in ApplicationRole return r

```
[
  {
    "_id": "ApplicationRole/946760",
    "_key": "946760",
    "_rev": "_UxXpBWG---",
    "claims": [],
    "name": "System Admin",
    "normalizedName": null
  }
]
```

The following problem occurs: [Screenshot?] 
ScreenShot: https://1drv.ms/i/s!AnU_xH1JBFXGhPN00nEe72sRU6KgJA
Interface Url is: http://localhost:8529/_db/_system/_admin/aardvark/index.html#collection/ApplicationRole/documents/1

(in Chrome console)
app.js?version=1488275232633:2 Uncaught TypeError: Cannot read property 'fullCount' of undefined
    at Object.success (app.js?version=1488275232633:2)
    at k (libs.js?version=1488275232633:2)
    at Object.fireWith [as resolveWith] (libs.js?version=1488275232633:2)
    at c (libs.js?version=1488275232633:3)
    at XMLHttpRequest.<anonymous> (libs.js?version=1488275232633:3)
success @ app.js?version=1488275232633:2
k @ libs.js?version=1488275232633:2
fireWith @ libs.js?version=1488275232633:2
c @ libs.js?version=1488275232633:3
(anonymous) @ libs.js?version=1488275232633:3

Clicking to main collections screen and back sometimes shows the data, but mostly not. I have to query to see the documents.

**Comment by hkernbach** [comment]
Thanks for the report. Unfortunately I am unable to reproduce. Could you please also add the response of the network request?

Url is like this (different id):
`http://localhost:8529/_db/_system/_api/job/149147678086235`

and appears when you navigate to: 
`http://localhost:8529/_db/_system/_admin/aardvark/index.html#collection/ApplicationRole/documents/1`

The content of the response should be something like this:
<details>
{"result":[{"_id":"ApplicationRole/946760","_key":"946760","_rev":"_Ux6Q4Ce---","claims":[],"name":"System Admin","normalizedName":null}],"hasMore":false,"cached":false,"extra":{"stats":{"writesExecuted":0,"writesIgnored":0,"scannedFull":1,"scannedIndex":0,"filtered":0,"httpRequests":0,"executionTime":3.190040588378906e-4},"warnings":[]},"error":false,"code":201}
</details>

**Comment by hkernbach** [comment]
Found it. Results slightly differ if it is a cached result. Fixed in 3.1 and devel.

**Comment by dothebart** [comment]
3.1.18 including this fix is now available for download.
